quickstart/docker/image/ziti-cli-functions.sh

```diff
@@ -279,6 +279,36 @@ function checkPrereqs {
   fi
 }
 
+function _portCheck {
+  if [[ "${1-}" == "" ]] || [[ "${2-}" == "" ]]; then
+    echo -e "_portCheck Usage: _portCheck <port> <portName>"
+    return 0
+  fi
+
+  envVar="${1-}"
+  if [[ -n "$ZSH_VERSION" ]]; then
+    envVarValue="${(P)envVar}"
+  elif [[ -n "$BASH_VERSION" ]]; then
+    envVarValue="${!envVar}"
+  else
+    echo -e "Unknown/Unsupported shell, cannot check port status"
+    return 1
```

Comment:
should this just return 0 in this case? i feel like you shouldn't fail for this particular reason. i'd just print a warning. i'd also print it in orange/yellow/red or something to get attention and say 'whatever, don't mess it up' from here.